src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -9853,7 +9853,7 @@ .maxstack  2
 ");
         }
 
-        [ConditionalFact(typeof(WindowsDesktopOnly), Reason = ConditionalSkipReason.EditAndContinueRequiresDesktop, AlwaysSkip = "https://github.com/dotnet/roslyn/issues/37047")]
+        [Fact]
```

Comment:
@tmat This test is failing in CI. I guess the merge is incorrect. Could you please take a look?

https://dev.azure.com/dnceng/public/_build/results?buildId=261145&view=logs

Reply:
Looking at the bug linked, seems this should be always skipped. Will fix